sdk/src/utils.ts

```diff
@@ -1,9 +1,15 @@
 import { ApiPromise, Keyring, WsProvider } from '@polkadot/api'
+import { ContractPromise } from '@polkadot/api-contract'
+import { WeightV2 } from '@polkadot/types/interfaces'
 import { IKeyringPair } from '@polkadot/types/types/interfaces'
 import { getSubstrateChain } from '@scio-labs/use-inkathon/chains'
 import { getBalance, initPolkadotJs as initApi } from '@scio-labs/use-inkathon/helpers'
 import { readFile } from 'fs/promises'
 import { Network } from './network.js'
+import { InvariantQuery, InvariantTx, PSP22Query, PSP22Tx, WrappedAZEROTx } from './schema.js'
+
+export const DEFAULT_REF_TIME = 100000000000
```

Comment:
I wonder if parameters needed for gas limits should be defined per contract, not globally